app/src/main/java/com/celzero/bravedns/ui/HomeScreenFragment.kt

```diff
@@ -551,21 +808,19 @@ class HomeScreenFragment : Fragment(R.layout.fragment_home_screen) {
      * When the application crashes/updates it goes into
      * red waiting state. This causes confusion to the users also requires
      * click of START button twice to start the app.
-     * FIX : The check for the controller state. If the state of the controller
-     * is activationRequested and the VPN is not connected then
+     * FIX : The check for the controller state. If persistence state has
+     * vpn enabled and the VPN is not connected then
      * the start will be initiated.
      */
     private fun maybeAutoStart() {
-        if (isVpnStarted && !VpnController.state().on) {
+        if (persistentState.getVpnEnabled() && !VpnController.state().on) {
```

Comment:
`isVpnStarted` is already pointing to the same presistentState var `activationRequested`, right? Why use `persistentState#getVpnEnabled`?

Reply:
done.